JsonClient/JsonClient/Program.cs

```diff
@@ -1,20 +1,27 @@
 ﻿using System;
 using System.Threading.Tasks;
-
+using JsonClient.Entities;
+using JsonClient.Attributes;
+using System.ComponentModel.DataAnnotations;
+using System.Collections.Generic;
+using System.Reflection;
 namespace JsonClient
 {
     class Program
     {
         static async Task Main(string[] args)
         {
            await runClient();
+
         }
 
-        static async Task runClient(){
+        static async Task runClient()
+        {
             
             
             int option = 0;
-             while(option < 7){
+             while(option < 7)
+             {
                 Console.WriteLine("Select an entity option:\n1 Comments\n2 Post\n3 Album\n4 Photo\n5 Todo\n6 User\n7 exit");
                 option = Int32.Parse(Console.ReadLine());

```

Comment:
Change Int32.Parse method to int.TryParse. Using the last method may avoid an exception.